repository/kafka/docs/latest/external-access.md

```diff
@@ -0,0 +1,262 @@
+
+![kudo-kafka](./resources/images/kafka-producer-consumer.png)
+
+To achieve this, we need to guarantee that the producers and consumers can directly access the brokers from inside and outside of a Kubernetes cluster. 
+
+## Internal access
+
+For the applications living inside the same kubernetes cluster. KUDO Kafka supports the headless service. 
+
+![kudo-kafka](./resources/images/internal-access.png)
+
+
+
+## External access
+
+For the producers and consumers living outside the Kubernetes cluster KUDO Kafka supports the Nodeport and Loadbalancer approach.
+
+#### Loadbalancer (recommended)
+
+Exposing the Apache Kafka brokers using the LoadBalancer Service Type is the recommended way for the KUDO Kafka.
+
+KUDO Kafka can be exposed easily for external access using Service Type [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer).
+
+The parameter used to configure the external loadbalancers is `EXTERNAL_ADVERTISED_LISTENER_TYPE`
+
+When set to `LoadBalancer` it creates one service per broker to expose each broker externally. 
+
+```
+kubectl kudo install kafka --instance=kafka \
+	-p EXTERNAL_ADVERTISED_LISTENER=true \
+	-p EXTERNAL_ADVERTISED_LISTENER_TYPE=LoadBalancer
+```
+
+![kudo-kafka-external-access-loadbalancers](./resources/images/external-access-loadbalancer.png)
+
+For these Kubernetes Services, KUDO Kafka uses the `externalTrafficPolicy:local` to avoid any second hop. As each broker gets its own loadbalancer we can avoid the extra hops and get a better network performance.
+
+```
+kubectl get svc
+NAME                     TYPE           CLUSTER-IP    EXTERNAL-IP                                                               PORT(S)                      AGE
+kafka-kafka-0-external   LoadBalancer   10.0.13.34    aebb0d2f2adda4b22b7d9b0c07a865b8-1700665253.us-west-2.elb.amazonaws.com   9097:31713/TCP               11s
+kafka-kafka-1-external   LoadBalancer   10.0.45.151   a8ae206a726b24fed91b82eb9110e0d1-1872607345.us-west-2.elb.amazonaws.com   9097:30786/TCP               11s
+kafka-kafka-2-external   LoadBalancer   10.0.32.219   a252e64a36f4443ecbea2fa4c4bf8441-1574422153.us-west-2.elb.amazonaws.com   9097:32048/TCP               11s
+```
+
+Producers and consumers can connect to `aebb0d2f2adda4b22b7d9b0c07a865b8-1700665253.us-west-2.elb.amazonaws.com:9097,a8ae206a726b24fed91b82eb9110e0d1-1872607345.us-west-2.elb.amazonaws.com:9097,a252e64a36f4443ecbea2fa4c4bf8441-1574422153.us-west-2.elb.amazonaws.com:9097` in the above case.
+
+To verify if the `server.properties` has the correct configuration for the `listeners`,`advertised.listeners` and `listener.security.protocol.map`, users can check the `server.properties` content in the broker pods. For example for broker `2` you just need to run the following command:
+
+```
+> kubectl exec -ti kafka-kafka-2 cat server.properties
```

Comment:
After the pvc monitoring PR was merged the kafka pod will two containers if metrics is enabled
```suggestion
> kubectl exec -ti kafka-kafka-2 -c k8skafka cat server.properties
```